Lombiq.Tests.UI/Services/UITestExecutor.cs

```diff
@@ -75,11 +60,6 @@ private static async Task ExecuteOrchardCoreTestInnerAsync(
         {
```

Comment:
The `_configuration.TestCancellationToken.IsCancellationRequested` can also stop the loop.